src/main/preload/callsWidget.js

```diff
@@ -1,94 +0,0 @@
-// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
There was no need for a separate calls widget script since we can assume that everything that is exposed is accessible remotely, and it was easier to just manage one for all use cases.